pyvelox/pyvelox.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include "pyvelox.h" // @manual
+#include "signatures.h" // @manual
```

Comment:
why do we need these @ manuals?

Reply:
I removed them for now ; but I do recall having build issues without them in fbcode (TA had similar ones for its bindings). Should that happen i will readd them.